filebeat/module/elasticsearch/ecs_log_samples/deprecation/runTask_deprecation.json

```diff
@@ -0,0 +1,8 @@
+{"@timestamp":"2020-01-09T17:12:15.385Z", "log.level": "WARN",  "message":"Field parameter [precision] is deprecated and will be removed in a future version.", "x-opaque-id":"asdf" , "service.name":"ES_ECS","process.thread.name":"elasticsearch[runTask-0][masterService#updateTask][T#1]","log.logger":"org.elasticsearch.deprecation.index.mapper.LegacyGeoShapeFieldMapper","type":"server","cluster.name":"runTask","node.id":"d-QKaUf1Q6Cv79ssDL5_CA","node.name":"runTask-0","cluster.uuid":"Iy7ZzpclS-2IPJT08MS8Cw"}
```

Comment:
All field names with dots should be replaced with nested fields.

Unless a "dedot" transformation is applied, these are not valid ECS documents.

Reply:
I think  this is not implemented in ecs-logging-java yet
https://github.com/elastic/ecs-logging-java/issues/51
Althought I feel like this is still under discussion? 
@felixbarny what do you think?

Reply:
Fields added by ecs-logging-java, like `process.thread.name` would be relatively simple to convert to nested objects. However, fields added by Elasticsearch, like `node.name` or things added to the MDC, like `"trace.id"`, can't easily be converted to nested objects.